.gitignore

```diff
@@ -54,3 +54,5 @@ Brewfile.lock.json
 
 # Visual Studio Code
 .vscode/
+
+tests/vendor
```

Comment:
Why are we adding this?

Reply:
I was trying to run `go mod tidy`, `go mod vendor` and then `go test` .
tests/vendor should not be committed.